userprofile/models.py

```diff
@@ -34,8 +34,7 @@ def get_by_request(cls, request):
         raise RuntimeError("Seeking user profile without authenticated user.")
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    # FIXME: refactor lang to selected_language which by default is blank
-    lang = models.CharField(max_length=5, default="en_US")
```

Comment:
How cool, this was added when the initial code was added to git in 2012... now after 8 years it actually got implemented :smiley: